x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/index.js

```diff
@@ -9,6 +9,8 @@ import { ColaGraph } from './views/cola_graph';
 import { DetailDrawer } from './views/detail_drawer';
 import { PropTypes } from 'prop-types';
 
+export { ConfigViewer } from './views/config_viewer';
```

Comment:
Are any changes to this file necessary in this PR? Particularly since I also found this: https://github.com/elastic/kibana/pull/18597/files#diff-a5e9d35eb329c11e0b107737f2c7d556R7